.travis.yml

```diff
@@ -54,6 +54,16 @@ matrix:
     - TEST_COMMAND="xvfb-run npm run test:integration"
 
   - node_js: '10'
+    if: type = pull_request
+    addons:
+      chrome: stable
+      firefox: latest
+    env:
+    - SKIP_LOCAL_BROWSERS=true
```

Comment:
If we are skipping local browsers here, then we don't need the addons here. That is the real bug: we expect the addons to be there, even though we explicitly skip them. As I said in the issue, we need to update `run-wct.sh` instead of fixing the Travis configuration.